addons/sale/migrations/12.0.1.1/openupgrade_analysis.txt

```diff
@@ -57,10 +57,9 @@ sale         / sale.order.line          / qty_delivered_manual (float)  : NEW ha
 sale         / sale.order.line          / qty_delivered_method (selection): NEW selection_keys: ['analytic', 'manual'], isfunction: function, stored
 sale         / sale.order.line          / untaxed_amount_invoiced (float): NEW isfunction: function, stored
 sale         / sale.order.line          / untaxed_amount_to_invoice (float): NEW isfunction: function, stored
-sale_order_dates / sale.order               / commitment_date (datetime)    : not stored anymore
-sale_order_dates / sale.order               / commitment_date (datetime)    : was renamed to expected_date [nothing to do]
+sale_order_dates / sale.order               / commitment_date (datetime)    : not a function anymore
 sale_order_dates / sale.order               / effective_date (date)         : module is now 'sale_stock' ('sale_order_dates')
-sale_order_dates / sale.order               / requested_date (datetime)     : was renamed to commitment_date [nothing to do]
+sale_order_dates / sale.order               / requested_date (datetime)     : DEL
```

Comment:
Why this change?

Reply:
I don't know.

Reply:
You haven't modified code that makes this change, isn't it?

Reply:
Exactly., thus it surprises me why this changed :S

Reply:
yes